draft-ietf-gnap-core-protocol.md

```diff
@@ -3409,6 +3422,8 @@ means, the AS SHOULD honor the revocation request to
 the token management URL as valid, since the end result is still the token
 not being usable.
 
+Note that some deployment decisions like self-contained stateless access tokens can make
```

Comment:
isn't it a bit redundant with what is already said above ? Note that stateless access tokens can make proactive
revocation difficult within a system, see {{security-stateless-tokens}}.

Reply:
I believe you're correct, this paragraph can be dropped.